src/Formats/StructuredFormatService.kt

```diff
@@ -1,6 +1,7 @@
 package org.jetbrains.dokka
 
 import java.util.LinkedHashMap
+import com.intellij.openapi.util.text.StringUtil
```

Comment:
It's from Kotlin compiler, so it will only be available in Kotlin's module, when we will split the project.